bundles/org.openhab.binding.coronastats/src/main/resources/ESH-INF/thing/channel-type.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="coronastats"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="count">

```

Comment:
Please create an individual channel type for each channel (and move the label texts from the thing definitions to here).

Reply:
Changed